rocky-raft/src/rocky/raft/dto/ConfigEntry.java

```diff
@@ -0,0 +1,40 @@
+package rocky.raft.dto;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class ConfigEntry implements Serializable {
+    private List<Address> currentConfig;
+    private List<Address> newConfig;
```

Comment:
Why do we need to differentiate? It's leader's job to first update config with union of old & new and the with new.